build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.build-logic.groovy-dsl-gradle-plugin.gradle.kts

```diff
@@ -50,11 +50,23 @@ tasks.withType<GroovyCompile>().configureEach {
 }
 
 tasks.withType<Test>().configureEach {
-    if (JavaVersion.current().isJava9Compatible) {
```

Comment:
The previous code assumed the Gradle build and the test were running on the same version